personal_assistant/menus.py

```diff
@@ -197,19 +196,19 @@ def notes_menu():
         elif command == "help":
             print(messages.notes_help_center)
 
-        elif command in ("add", "create", "touch"):
+        elif command in ("add-note"):
```

Comment:
why would we add to commands '-note' suffix if we are in notes menu?

Reply:
this is the name of the umbrella term for different utterances that mean add a new note. Needed to distinguish from the add-contact phrases